### PR TITLE
Issue-1174: Remove flake8-coding

### LINKED ISSUE #1174: Remove flake8-coding
We need to vendor this dependency to our own codebase:
- create new violation
- create new logical check
- write tests
- copy configuration

Implementation: https://github.com/tk0miya/flake8-coding/blob/master/flake8_coding.py#L52-L73

Related: #1139

### PR DESCRIPTION
**Pull request by Kvm99**
# I have made things!

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [ ] I have created at least one test case for the changes I have made
- [ ] I have updated the documentation for the changes I have made
- [x] I have added my changes to the `CHANGELOG.md`

## Related issues

- Closes #1174
- Refs #1139

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,10 @@ Semantic versioning in our case means:
 - Adds baseline information for all complexity violation messages: `x > baseline`
 - Changes how cognitive complexity is calculated
 - Adds support for positional arguments in different checks
+- Forbids to use `continue` and `break` in `finally`. It is a terrible practice, because
```
Comment (sobolevn) [inline CHANGELOG.md]: This change is unrelated. Please, make sure you didn't commit extra files (there are more than 300 files changed!)

**Comment by radzak** [comment]
@sobolevn I believe these lines could also be removed from https://github.com/wemake-services/wemake-django-template right?

**Comment by sobolevn** [comment]
@radzak yes, please! Would you send a PR?
